cross/ffmpeg/Makefile

```diff
@@ -5,12 +5,33 @@ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = http://www.ffmpeg.org/releases/
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/x264 cross/lame
+DEPENDS = cross/x264 cross/lame cross/fdk-aac cross/speex cross/flac cross/libvorbis 
+DEPENDS += cross/openjpeg cross/openssl cross/freetype cross/libass cross/libbluray 
+DEPENDS += cross/libopus cross/libtheora cross/rtmpdump cross/libvpx
 
 HOMEPAGE = http://www.ffmpeg.org/
 COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec - the leading audio/video codec library
 LICENSE  =
 
-CONFIGURE_ARGS = --disable-yasm --enable-gpl --enable-libx264 --enable-libmp3lame --enable-nonfree --enable-cross-compile --arch=$(ARCH) --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX) --extra-cflags="-I$(WORK_DIR)/install$(INSTALL_PREFIX)/include" --extra-ldflags="-L$(WORK_DIR)/install$(INSTALL_PREFIX)/lib" --extra-libs=-ldl
+CONFIGURE_ARGS = --disable-yasm --disable-asm --enable-gpl --enable-version3 --enable-nonfree --enable-cross-compile --enable-rpath --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX) --extra-cflags="-I$(WORK_DIR)/install$(INSTALL_PREFIX)/include" --extra-ldflags="-L$(WORK_DIR)/install$(INSTALL_PREFIX)/lib" --extra-libs="-lxml2 -ldl"
```

Comment:
I know this is a WIP, but I noticed this in scrolling past. Two suggestions for the final product:
- Don't use `--disable-asm` if it can be avoided, you'll probably see a huge performance drop when using ffmpeg. There _might_ be a couple arches that don't support it (older ARM and PPC maybe), but that should be tested.
- With the addition of `native/yasm`, you can (should ;)) use that for ffmpeg too.